src/swupd/manifest.go

```diff
@@ -0,0 +1,117 @@
+package swupd
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// ManifestHeader contains metadata for the manifest
+type ManifestHeader struct {
+	Format      uint16
+	Version     uint32
+	Previous    uint32
+	FileCount   uint32
+	TimeStamp   uint64
```

Comment:
It would be more normal to use time.Time for the timestamp. See [Fileinfo](https://golang.org/pkg/os/#FileInfo) from the standard library